pyproject.toml

```diff
@@ -41,6 +41,23 @@ eval-type-backport = "0.2.0"
 curl_cffi = "0.6.2"
 pandoc = "*"
 pillow = "^10.2.0"
+requests-oauthlib = ">=1.3.1"
+
+# Type stubs for dependencies
+types-redis = "^4.6.0.20240106"
+types-requests = "^2.31.0.20240106"
+types-python-dateutil = "^2.8.19.20240106"
+types-pytz = "^2024.1.0.20240104"
+types-PyYAML = "^6.0.12.12"
+types-beautifulsoup4 = "^4.12.0.20240106"
+types-colorama = "^0.4.15.20240106"
+types-docutils = "^0.20.0.20240106"
+types-jsonschema = "^4.21.0.20240106"
+types-pillow = "^10.2.0.20240106"
+types-protobuf = "^4.24.0.20240106"
+types-setuptools = "^69.0.0.20240106"
+types-six = "^1.16.21.20240106"
+types-urllib3 = "^1.26.25.14"
```

Comment:
why we need to add this?

Reply:
I added this last time because I cannot pass the pre-commit check😢